server/internal/config/config.go

```diff
@@ -5,40 +5,44 @@ import (
 	"os"
 	"path/filepath"
 	"strings"
+	"time"
 
 	"github.com/ark-network/ark/common"
 	"github.com/spf13/viper"
 )
 
 type Config struct {
-	Datadir               string
-	WalletAddr            string
-	RoundInterval         int64
-	Port                  uint32
-	EventDbType           string
-	DbType                string
-	DbDir                 string
-	DbMigrationPath       string
-	SchedulerType         string
-	TxBuilderType         string
-	BlockchainScannerType string
-	NoTLS                 bool
-	NoMacaroons           bool
-	Network               common.Network
-	LogLevel              int
-	RoundLifetime         int64
-	UnilateralExitDelay   int64
-	BoardingExitDelay     int64
-	EsploraURL            string
-	NeutrinoPeer          string
-	BitcoindRpcUser       string
-	BitcoindRpcPass       string
-	BitcoindRpcHost       string
-	TLSExtraIPs           []string
-	TLSExtraDomains       []string
-	UnlockerType          string
-	UnlockerFilePath      string
-	UnlockerPassword      string
+	Datadir                 string
+	WalletAddr              string
+	RoundInterval           int64
+	Port                    uint32
+	EventDbType             string
+	DbType                  string
+	DbDir                   string
+	DbMigrationPath         string
+	SchedulerType           string
+	TxBuilderType           string
+	BlockchainScannerType   string
+	NoTLS                   bool
+	NoMacaroons             bool
+	Network                 common.Network
+	LogLevel                int
+	RoundLifetime           int64
+	UnilateralExitDelay     int64
+	BoardingExitDelay       int64
+	EsploraURL              string
+	NeutrinoPeer            string
+	BitcoindRpcUser         string
+	BitcoindRpcPass         string
+	BitcoindRpcHost         string
+	TLSExtraIPs             []string
+	TLSExtraDomains         []string
+	UnlockerType            string
+	UnlockerFilePath        string
+	UnlockerPassword        string
+	FirstMarketHour         int64
+	MarketHourPeriod        int64
+	MarketHourRoundLifetime int64
```

Comment:
What's this? Also, _RoundLifetime_ is not correct (where used elsewhere has to be replaced)

Reply:
Done